packages/grant-explorer/src/features/api/rounds.ts

```diff
@@ -65,7 +65,7 @@ async function fetchRoundsByTimestamp(
         round.roundMetaPtr.pointer
       );
       round.roundMetadata = roundMetadata;
-      round.chainId = chainId;
+      round.chainId = chainId.toFixed(0).toString();
```

Comment:
Two questions pop to mind when seeing this: 1) "is `chainId` a float that it needs a `toFixed`?" and 2) what is the purpose of `toString()` when `toFixed()` returns a string?

Reply:
the ToString is indeed superfluous here. this is a workaround for a behaviour of TS enums where if you call toString() on an instance of it, it will output the string value (OPTIMISM_MAINNET) instead of the stringified number ("10"). this is a common gotcha, started a thread in eng. to see how we can improve this.

Reply:
removed the toString